pkgs/development/libraries/boost/generic.nix

```diff
@@ -114,6 +116,7 @@ stdenv.mkDerivation {
   patchFlags = [];
 
   patches = patches
+  ++ optional stdenv.hostPlatform.isMips64 ./mips64-context.patch
```

Comment:
I think we should always apply this to make sure the patch works in the future and less maintenance is on you. If you take a look at the other patches they are mostly specific to boost version ranges.

Also where did you find the patch? Is there a source we can mention?